ops/terraform/services/base/ephemeral.tf

```diff
@@ -89,11 +96,8 @@ data "aws_db_cluster_snapshot" "seed" {
 
   db_cluster_identifier = "bfd-${local.seed_env}-aurora-cluster"
   most_recent           = true
-  db_cluster_snapshot_identifier = lookup(
-    local.common_nonsensitive_ssm,
-    "/bfd/${local.env}/common/nonsensitive/rds_snapshot_identifier",
-    var.ephemeral_rds_snapshot_id_override
-  )
+  # default to latest snapshot if no override is provided
+  db_cluster_snapshot_identifier = var.ephemeral_rds_snapshot_id_override
```

Comment:
@aschey-forpeople  I'm trying to remember this. Is this when it looks up the snapshot but only the last 3 snapshots are kept? Wouldn't the default be too old in some cases like how it was for us when we tested?